upbound_uxp/manifest.json

```
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f1427ca4-4b88-484a-a7c5-9cc66542b0d9",
+  "app_id": "upbound-uxp",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "Upbound UXP",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "Upbound UXP",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "upbound_uxp.",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "dashboards": {
+	"upbound_uxp_overview": "assets/dashboards/upbound_uxp_overview.json",
+	"upbound_uxp_min_set": "assets/dashboards/upbound_uxp_min_set.json"
+    },
+    "monitors": {},
+    "logs": {},
+    "saved_views": {}
+  },
+  "author": {
+    "support_email": "markus.schweig@upbound.io",
+    "name": "Markus Schweig",
```

```suggestion
    "name": "Upbound",
```
This should be the company's name

---

changed to Upbound